sonar-project.properties

```diff
@@ -0,0 +1,33 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=owncloud_data_exporter
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/data_exporter
+sonar.links.ci=https://drone.owncloud.com/owncloud/data_exporter/
+sonar.links.scm=https://github.com/owncloud/data_exporter
+sonar.links.issue=https://github.com/owncloud/data_exporter/issues
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+# Just look in these directories for code
+sonar.sources=.
+sonar.inclusions=appinfo/**,lib/**
+
+# Pull Requests
+sonar.pullrequest.provider=GitHub
+sonar.pullrequest.github.repository=owncloud/core
+sonar.pullrequest.base=${env.SONAR_PULL_REQUEST_BASE}
+sonar.pullrequest.branch=${env.SONAR_PULL_REQUEST_BRANCH}
+sonar.pullrequest.key=${env.SONAR_PULL_REQUEST_KEY}
+
+# Properties specific to language plugins:
+sonar.php.coverage.reportPaths=results/clover-phpunit-php7.2-mariadb10.2.xml,results/clover-phpunit-php7.2-mysql5.5.xml,results/clover-phpunit-php7.2-mysql5.7.xml,results/clover-phpunit-php7.2-postgres9.4.xml,results/clover-phpunit-php7.2-oracle.xml,results/clover-phpunit-php7.3-mysql5.5.xml,results/clover-phpunit-php7.4-mysql5.5.xml,results/clover-phpintegration-php7.2-sqlite.xml,results/clover-phpintegration-php7.2-mariadb10.2.xml,results/clover-phpintegration-php7.2-mysql5.5.xml,results/clover-phpintegration-php7.2-mysql5.7.xml,results/clover-phpintegration-php7.2-postgres9.4.xml,results/clover-phpintegration-php7.2-oracle xml,results/clover-phpintegration-php7.3-mysql5.5.xml,results/clover-phpintegration-php7.4-mysql5.5.xml
```

Comment:
https://drone.owncloud.com/owncloud/data_exporter/1406/22/7
There are some `WARN: PHPUnit xml coverage report not found: results/clover-phpunit-php7.3-mysql5.5.xml`
Those need to be removed from this list.
https://drone.owncloud.com/owncloud/data_exporter/1406/22/6 has a list of what is there:
```
+ ls -l results
total 1128
-rw-r--r-- 1 root root 96362 Nov 24 10:06 clover-phpintegration-php7.2-mariadb10.2.xml
-rw-r--r-- 1 root root 96362 Nov 24 10:06 clover-phpintegration-php7.2-mysql5.5.xml
-rw-r--r-- 1 root root 96362 Nov 24 10:06 clover-phpintegration-php7.2-mysql5.7.xml
-rw-r--r-- 1 root root 96362 Nov 24 10:06 clover-phpintegration-php7.2-oracle.xml
-rw-r--r-- 1 root root 96362 Nov 24 10:06 clover-phpintegration-php7.2-postgres9.4.xml
-rw-r--r-- 1 root root 96362 Nov 24 10:06 clover-phpintegration-php7.2-sqlite.xml
-rw-r--r-- 1 root root 90346 Nov 24 10:06 clover-phpunit-php7.2-mariadb10.2.xml
-rw-r--r-- 1 root root 90346 Nov 24 10:06 clover-phpunit-php7.2-mysql5.5.xml
-rw-r--r-- 1 root root 90346 Nov 24 10:06 clover-phpunit-php7.2-mysql5.7.xml
-rw-r--r-- 1 root root 90346 Nov 24 10:06 clover-phpunit-php7.2-oracle.xml
-rw-r--r-- 1 root root 90346 Nov 24 10:06 clover-phpunit-php7.2-postgres9.4.xml
-rw-r--r-- 1 root root 90346 Nov 24 10:06 clover-phpunit-php7.2-sqlite.xml
```
Note: not all the PHP unit and integration test pipelines have coverage enabled.